pkg/machinepool/machinepool.go

```diff
@@ -957,18 +957,19 @@ func (m *machinePool) CreateNodePools(r *rosa.Runtime, cmd *cobra.Command, clust
 	sumOfMaxReplicas := maxReplicas
 	sumOfMinReplicas := minReplicas
 
-	for _, np := range cluster.NodePools().Items() {
+	nodepools, err := r.OCMClient.GetNodePools(cluster.ID())
+
+	for _, np := range nodepools {
 		// If autoscaling, calculate min and max, use min and max in separate messages below
-		autoscaling, ok := np.GetAutoscaling()
-		if !ok || autoscaling == nil {
+		npAutoscaling, ok := np.GetAutoscaling()
+		if ok {
```

Comment:
confused here
should we get  min/maxReplicas when autoscaling enabled, and get fixed replicas if not?
in current code, you will try to get all of them when autoscaling is enabled, and do nothing if not.